README.md

```diff
@@ -70,29 +70,32 @@ The cron format consists of:
 │    └──────────────────── minute (0 - 59)
 └───────────────────────── second (0 - 59, OPTIONAL)
 ```
+CRON has 5 params, while node-schedule has 6.  The CRON lowest resolution is limited to 1 minute, while node-schedule resolution goes down to 1 second.
 
```

Comment:
Maybe make a reference to `cron-parser`, the module we're using for cron syntax handling.